src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,7 @@
+@function getHorizontalShift($container-size, $element-width) {
+  @return $container-size / 2 - $element-width / 2;
+}
+
+@function getVerticalShift($container-size, $element-height) {
+  @return $container-size / 2 - $element-height;
+}
```

Comment:
But it's a same logic, why duplicate it?

Reply:
it had to be like:
@function getShift($container-size, $element-side, $divider) {
  @return $container-size / 2 - $element-side / $divider;
}

then we call it like - getShift($watch-size, $minute-hand-width, 2)

@OleksiiBlezniuk right?